docs/gethue/content/en/posts/2021-04-09-collaborate-on-your-sql-queries-and-results-directly-within-slack.md

```diff
@@ -57,33 +57,33 @@ What could be better than improving the collaboration by using one of the best c
 
 Although currently in **beta**, this app integration expands the ability to share query links or gists to the desired Slack channels which then unfurls in a rich preview for other users. It even gives you the result file if the result has not expired.
 
-![Share to Slack Architecture]()
+![Share to Slack Architecture](https://cdn.gethue.com/uploads/2021/04/share_to_slack_architecture.png)
 
 The main bot logic lies on top of the Hue server listening to the events posted by the Hue App from Slack to an endpoint, processing the data accordingly and using Slack API methods for sending the desired response.
 
 To manually set-up the Hue App, follow the steps mentioned in the [link here]()
 
 Now, open Hue, run some query and copy its link
 
-![Run Query in Hue]()
+![Run Query in Hue](https://cdn.gethue.com/uploads/2021/04/run_query_in_hue.png)
 
 Paste it in the Slack channel and see the unfurl magic!
 
-![Query Link Preview]()
+![Query Link Preview](https://cdn.gethue.com/uploads/2021/04/query_link_preview.png)
 
 ### Wonder why the query result (if available) is shown in this way?
```

Comment:
FYI: pretty long title (does not always show up well)

Reply:
Reduced the heading a bit